imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -63,47 +79,57 @@ class CaseExplorer extends Component {
   }
   render () {
     const { isLoading, dispatch, match } = this.props
-    const { unitsDict } = this.state
-
+    const { unitsDict, status, expandedUnits } = this.state
     return (
       <div className='flex flex-column roboto overflow-hidden flex-grow h-100'>
         <div className='bb b--black-10 overflow-auto flex-grow'>
+          <div className='flex pl3 pv3 bb b--very-light-gray'>
+            <div onClick={() => this.handleStatusClicked(0)} className={'f6 fw5 ' + (status === 0 ? 'mid-gray' : 'silver')}> Open </div>
+            <div onClick={() => this.handleStatusClicked(1)} className={'f6 fw5 ml4 ' + (status === 1 ? 'mid-gray' : 'silver')}> Closed </div>
+            {/* <div onClick={() => this.handleStatusClicked(2)} className={'f6 fw5 ml4 ' + (status === 2 ? 'mid-gray' : 'silver')}> Assigned To</div> */}
+          </div>
           {!isLoading && Object.keys(unitsDict).map(unitTitle => {
```

Comment:
You should change `map` to `reduce` or use `filter` before map, instead of returning `undefined` in L#97

Reply:
@nbiton Should this be done using actions and reducers?